identity/client/src/pages/users/modals/EditModal.tsx

```diff
@@ -39,34 +41,32 @@ const EditModal: FC<UseEntityModalProps<User>> = ({
       loadingDescription={t("Updating user")}
       confirmLabel={t("Update user")}
     >
+      <TextField
+        label={t("Username")}
+        value={user.username}
+        placeholder={t("Enter username or user ID")}
```

Comment:
💭 As in another comment, I know this is part of the designs but I'm not sure it makes sense for it to suggest `user ID`, we use the term `username` in other instances including the API spec so `user ID` here doesn't add anything IMO

Reply:
FYI @gastonpillet01, I adjusted this as well.